### PR TITLE
Add support for custom MyGet tenants

### PR DESCRIPTION
**Pull request by maartenba**
Add support for custom MyGet tenants (e.g. dotnet.myget.org)

Sample shield URL would be: `https://img.shields.io/dotnet.myget/dotnet-coreclr/v/Microsoft.DotNet.CoreCLR.svg`

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -3435,6 +3435,16 @@ mapNugetFeed('myget\\/(.*)', 1, function(match) {
   };
 });
 
+// MyGet (custom tenant, e.g. dotnet.myget.org)
+mapNugetFeed('(.*)\.myget\\/(.*)', 1, function(match) {
```
Comment (weshaggard) [inline server.js]: Should this be folded into the existing myget mapping?

**Comment by weshaggard** [comment]
Thanks @maartenba for submitting this PR. We (dotnet\corefx) team would like to use these badges for some of our packages.

**Comment by espadrine** [comment]
@maartenba Thanks for your contribution! I agree for the record with @weshaggard. Can you perform this change to the existing myget mapping? Would you be against me rewriting your commit?

@weshaggard On an unrelated topic, I would love it if you had a bit of time to look at #678.

**Comment by maartenba** [comment]
@espadrine Feel free to rewrite, totally fine with that :-)

File: server.js (hunk 1 above)
Comment (espadrine) [inline server.js]: Also, the `\` needs to be escaped, and the 1 turned into a 2 (as there is one more regex grouping).

**Comment by espadrine** [comment]
Thanks for the contribution. The change is live! You can try it out; I'll add an example to the main website in 24h.

**Comment by maartenba** [comment]
Awesome, thanks!

**Comment by weshaggard** [comment]
Thanks for the taking the change.

>@weshaggard On an unrelated topic, I would love it if you had a bit of time to look at #678.

Sorry I don't have the much influence or context on the nuget team. It looks like you have filed the right issue on nuget\home.

**Pull request closed without merging**